.claude/skills/onboard-component-from-jira.md

```
@@ -0,0 +1,250 @@
+---
+skill: onboard-component-from-jira
+description: Automates onboarding of a new component to the MCE operator from a Jira ticket
+---
+
+# Onboard Component from Jira
+
+Automates the onboarding of a new component to the Multicluster Engine (MCE) operator by reading a component template from a Jira ticket description and making all necessary code changes.
+
+## Usage
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)